raiden/exceptions.py

```diff
@@ -189,6 +189,10 @@ class DuplicatedChannelError(RaidenRecoverableError):
     """Raised if someone tries to create a channel that already exists."""
 
 
+class ChannelNotOpenError(RaidenRecoverableError):
```

Comment:
Nitpick: I would generalize this to something like `InvalidChannelState`

Reply:
Hmmm perhaps unexpected channel state? Invalid would mean that it's something which we can't even parse.